### PR TITLE
nsc-events-nextjs_8_392_change-password-sign-out

### LINKED ISSUE #392: Bug: Changing password should sign out user when redirecting to sign in page
After successfully changing password, the user should be signed out when redirected to the Sign In page.

### PR DESCRIPTION
**Pull request by intisarosman1**
Resolves #392

This PR fixes the bug where the user successfully changing their password redirects them to the sign-in page, but they are still logged in. Now, changing your password should sign you out, so that you can log in with your new password.

Recreate the bug:
- While logged in, head to the Change Password page (auth/change-password)
- Enter previous password and new password and confirm
- You should be redirected to the Sign-in page, but in the Navbar, it shows "Sign out", proving that you are still signed in

Please ensure that you are signed in and enter the correct previous password. I'm unsure of why there are some people experiencing issues with it, but the change password functionality is outside of the changes of my PR; I only made it so that it logs out after the change is successful. 

https://github.com/SeattleColleges/nsc-events-nextjs/assets/77607212/824663af-e9d5-4a5d-9a1e-863166f703bd

### CODE REVIEW

**Review by el-riber — COMMENTED** [review]
the page is working well, but the functionality you described I didn't try as I can't signup or log in, maybe I'm testing it wrong

Screenshot 2024-06-12 at 10 19 33 AM


Screenshot 2024-06-12 at 10 19 43 AM

**Review by alasali1 — COMMENTED** [review]
I'm not sure why, but I'm getting this problem 

https://github.com/SeattleColleges/nsc-events-nextjs/assets/77362461/938f275f-f745-43a6-9d7d-e5f98b69e3ef

The password change is failing whether using a user, admin, or creator account.

**Comment by taylorpapke** [comment]
> I'm not sure why, but I'm getting this problem
> 
>  NSC.Events.Mozilla.Firefox.2024-06-13.01-29-44.mp4 
> 
> The password change is failing whether using a user, admin, or creator account.

I'm also getting this error.

**Review by heosman — APPROVED** [review]
I used to get the error that everyone else was experiencing, but strangely I'm not anymore. It might have been because I was putting in the wrong password for the current password field. When I first tested this PR and I was experiencing the error, I checked if this was also happening on the main branch, and it was, so I don't think it's because of this PR's changes. I'm also running the frontend and backend locally, if that makes a difference.

https://github.com/SeattleColleges/nsc-events-nextjs/assets/77591083/9920f0f7-62d3-4d37-898a-41e9cf335706

File: app/auth/change-password/page.tsx
Comment (gitbiruk2010) [inline app/auth/change-password/page.tsx:1]: Good job addressing that!

**Review by taylorpapke — APPROVED** [review]

**Pull request merged**